README.md

```diff
@@ -259,8 +259,7 @@ RUN add-tool-shed --url 'http://testtoolshed.g2.bx.psu.edu/' --name 'Test Tool S
 RUN install-biojs msa
 
 # Install deepTools
-RUN install-repository \
-    "--url https://toolshed.g2.bx.psu.edu/ -o bgruening --name deeptools"
+RUN install-tools my_tool_list.yml
```

Comment:
Ok, fair enough let's leave this like it is. This sounds really awkward, especially because the playbook -> role split.
Thanks @afgane